backend/models/userModel.ts

```diff
@@ -10,7 +10,8 @@ const userSchema = new mongoose.Schema(
     email: {
       type: String,
       required: true,
-      unique: true
+      unique: true,
+      lowercase: true
```

Comment:
Question: does this force any user-entered string to be lowercase or does it fail validation if the user enters an email with capitals? If it's the second option we should consider forcing it to be lowercase rather than throwing an error.

Reply:
From my testing, it seems to force any user-entered string to be lowercase, it doesn't fail validation.